### ISSUE TITLE
Not showing errors on WSL

### ISSUE THREAD

**Issue by sanzoghenzo** [mxab/pre-commit-trivy#19]
Hi there!
I'm probably doing something wrong, but I can't get trivy error messages when running pre-commit (either via pre-commit run --all or actual pre-commit hook).

It outputs

```
trivyfs-docker...........................................................Failed
- hook id: trivyfs-docker
- exit code: 1
```

And nothing more.

This is my pre-commit config:

```yaml
repos:
  - repo: https://github.com/mxab/pre-commit-trivy.git
    rev: v0.12.0
    hooks:
      - id: trivyfs-docker
        args: [.]
      - id: trivyconfig-docker
        args: [.]
```

Using trivy, directly or via docker container, it correctly shows the errors.

I'm running it on Windows 10 inside an Ubuntu WSL.

Is there something I'm missing here?

Thanks for your attention

**Comment by mxab** [comment]
hmm can you try `args: ['.']` with the `'`
or 
```
args:
 - .
 ```

**Comment by sanzoghenzo** [comment]
Hi @mxab , that won't make yamlfix happy 😅 

Jokes aside, i tried it and it doesn't make a difference (it is valid yaml after all).

I also tried with the `[--skip-dirs, ./tests, .]` version, even if I don't have that folder in this specific project, and I got the same result.

It might be a WSL problem, since it works as expected on windows.

It may be also due to the fact that I use Rancher Desktop instead of Docker Desktop; I'll check with other docker-based pre-commit hooks to see if the problem is consistent
